src/App.css

```
@@ -1,34 +1,34 @@
 * {
-  font-family: 'Nunito', sans-serif;
+  font-family: "Nunito", sans-serif;
```

Stylelint (17.13.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)